code/datums/candidate_poll.dm

```diff
@@ -33,7 +33,6 @@
 	return ..()
 
 /datum/candidate_poll/Destroy()
-	QDEL_NULL(alert_button)
```

Comment:
What happened to this? What happens to the buttons when the poll ends? 

Doesn't this still need reference handling? Do we need `QDEL_LIST(linked_alert_buttons)`?

Reply:
@MrMelbert the buttons have their own timeout (which matches the poll duration var) and they go away on their own

Reply:
But how do we handle that reference here? We can't just keep them in this list or they will hard delete.

Reply:
done